WasatchNET/Spectrometer.cs

```diff
@@ -2244,8 +2285,15 @@ public bool sendSWTrigger()
         /// If the caller doesn't want to block on this, they can always change
         /// integrationTimeMS within a Task.Run closure.
         /// </remarks>
+        ///
+        /// <todo>
+        /// Consider simply disabling this feature (returning from this function)
+        /// if autoTrigger is disabled or using HW triggering.  In such cases, if
+        /// the user wants a throwaway, they can generate it themselves.
+        /// </todo>

```

Comment:
WPSC was already performing this in software. I'm admittedly not very familiar with our various triggering setups